source/migration.txt

```diff
@@ -0,0 +1,176 @@
+  can initiate live migration. Users authenticate to {+clusters+} using 
+  :manual:`SCRAM-SHA-1 or SCRAM-SHA-256 </core/security-scram/>`.
+
+- Live migration automates most tasks. For the fully-managed "pull" and "push" methods, live migration 
+  monitors key metrics, provisions the host servers and the strict sequencing of the migration commands. 
+  You specify the resource requirements and scalable options to prevent over-provisioning.
+
+- Detailed instructions help you provision migration hosts and scale destination {+clusters+} 
+  to control costs. Recommendations include appropriate {+cluster+} sizing 
+  and temporary scaling, followed by resizing to optimal levels post-migration.
+
+- Live migration uses |mongosync| to facilitate fast cutover through parallel data copying.
+  Processes manage temporary network interruptions and {+cluster+} elections, using continuous 
+  data synchronization and a final cutover phase to achieve minimal downtime. Retry 
+  mechanisms and pre-migration validations enhance resilience against interruptions.
+
+- Monitor migrations with real-time status updates and notifications.
+
+Live Migration Methods
+----------------------
+
+You can use |com| to push data into |service| or use a live migration server 
+to pull data into |service|.
+
+Ensure you allocate adequate |cpu| and network resources for the migration host. 
+While you can run multiple concurrent migrations, each deployment must have a 
+dedicated migration host.
+
+All live migration methods require that the source and destination databases run 
+MongoDB 6.0.13+ or MongoDB 7.0.8+. To migrate data from databases using prior 
+versions of MongoDB, see `Legacy Migration <https://www.mongodb.com/docs/atlas/legacy-migration/>`__ or :ref:`arch-center-manual-migration`. 
+
+* **Pull data into Atlas.** |service| pulls data from the source MongoDB deployment 
+  and requires access to the source deployment through the deployment's firewall. When the 
+  {+clusters+} are nearly synced, you must stop write operations on the source, 
+  redirect applications to the |service| {+cluster+} and restart them. The following
+  considerations apply:
+
+  - Best for deployments not monitored by |com|. 
+  - The source database must be publicly accessible to allow inbound access from the live migration server.
+  - Doesn't support :ref:`VPC peering <vpc-peering>` or :ref:`private endpoints <private-endpoint>`
+    for either the source or destination {+cluster+}.
+  - Source and destination {+cluster+} topologies must match. For example, both 
+    must be replica sets or sharded clusters with the same number of shards.
+  - Plan for minimal downtime to stop writes and restart applications with a new connection string. 
+    The migration process is CPU-intensive and requires significant network bandwidth. MongoDB 
+    recommends that the migration host(s) are run on their own dedicated server with a high 
```

Comment:
avoid using (s) to indicate an optional plural https://www.mongodb.com/docs/meta/style-guide/style/plurals/
```suggestion
    recommends that migration hosts are run on their own dedicated server with a high 
```